npm/design-system/src/components/FileExplorer/FileExplorer.module.scss

```diff
@@ -0,0 +1,73 @@
+@use '../../index.scss' as *;
+
+.nav {
+  user-select: none;
+  white-space: nowrap;
+}
+
+.ul {
```

Comment:
Just curios why classes are named like html tags, looks a little confusing as for me.